models/POP/shell_scripts/cesm2_1/DART_params.csh

```diff
@@ -0,0 +1,216 @@
+
+# ==============================================================================
+# Make sure the CESM directories exist.
+# VAR is the shell variable name, DIR is the value
+# ==============================================================================
+ 
+foreach VAR ( cesmroot dartroot stagedir )
+   set DIR = `eval echo \${$VAR}`
+   if ( ! -d $DIR ) then
+      echo "ERROR: directory '$DIR' not found"
+      echo " In the setup script check the setting of: $VAR "
+      exit -1
+   endif
+end
+
+# ==============================================================================
+# OSSE/Perfect Model experiments only.
+# If there is an ensemble of POP states to choose from, which one do you want as
+# the truth? There is an argument for picking an instance that will not be part
+# of the ensemble used for the assimilation experiment. 
+
+setenv SingleInstanceRefcase 0
+setenv TRUTHinstance 23
+
+# ==============================================================================
+#
+# stop_option   Units for determining the forecast length between assimilations
+# stop_n        Number of time units in each forecast
+# resubmit      How many job steps to run on continue runs (should be 0 initially)
+
+setenv resubmit     0
+setenv stop_option  ndays
+setenv first_STOP_N 3
+setenv STOP_N       1
+setenv resubmit     0
+
+# ==============================================================================
+# Settings for the data atmosphere
+
+setenv stream_year_align 1998
+setenv stream_year_first 1998
+setenv stream_year_last  2010
+
+setenv short_term_archiver off
+
+# ==============================================================================
+# job settings and machine-specific commands:
+#
+# mach            Computer name
+# ACCOUNT         Project that hours will be charged to
+# queue           can be changed during a series by changing the ${CASE}.run
+# timewall        can be changed during a series by changing the ${CASE}.run
+# ==============================================================================
+
+setenv mach         cheyenne
+setenv ACCOUNT      P93300612
+setenv queue        regular
+setenv timewall     00:15:00
+
+# ==============================================================================
+# by setting the values above you should be able to execute this script and
+# have it run.  however, for running a real experiment there are still many
+# settings below this point - e.g. component namelists, history file options,
+# the processor layout, xml file options, etc - that you will almost certainly
+# want to change before doing a real science run.
+# ==============================================================================
+
+# The CESM compile step takes enough resource that cheyenne requires a wrapper
+# If your platform does not have this restriction, set BUILD_WRAPPER to '' 
+# setenv BUILD_WRAPPER ''
+setenv BUILD_WRAPPER "qcmd -q share -l select=1 -A ${ACCOUNT} --"
+setenv nodes_per_instance 2
```

Comment:
nodes_per_instance is set here and in the setup scripts.

Reply:
Removed it from DART_params.csh and left it in the setup scripts.

Reply:
Sorry ... it looks like `nodes_per_instance` still exists in DART_params.csh ... the other thing is that setting `nodes_per_instance` here is kinda the wrong place, as people might think it has something to do with the resources needed to build CESM - which is not true.

Reply:
Yep, I haven't pushed to my fork yet. Apologies, I have been marking them as resolved when I've made the commit locally as a way of checking off the boxes as I'm processing of the revisions. I can go back and mark them as unresolved after I've pushed to my fork.